hi.js

```diff
@@ -1,821 +1,821 @@
-windows.chall.customs चुनौतियों को कस्टम खेलो में पूरा नहीं किया जा सकता
-twitch.link क्या आप वास्तव में अपना Twitch अकाउंट लिंक करना चाहते है?
-twitch.unlink क्या आप वास्तव में अपना Twitch अकाउंट अनलिंक करना चाहते है?
-windows.clan.leave क्या आप वास्तव में क्लैन को छोड़ना चाहते हैं?
-windows.clan.delete क्या आप वास्तव में क्लैन और उसके डेटा को मिटाना चाहते है?
-windows.clan.kick क्या आप वास्तव में {0} को क्लैन से निकालना चाहते हैं?
-windows.store.premium.purchase क्या आप वास्तव में {0} दिनों के लिए Premium खरीदने वाले हैं?
-windows.store.premium.gift Are you sure you want to Gift '{0}' {1} days of Premium?
-windows.playerlist.ban क्या आप वास्तव में {0} को खेल से बैन करने वाले है?
-windows.playerlist.flag क्या आप वास्तव में {0} को हैकरटैग चाहते वाले हैं?
-windows.playerlist.unflag क्या आप वास्तव में {0} का हैकरटैग हटाने चाहते हैं?
-windows.playerlist.punish क्या आप वास्तव में {0} पर कार्रवाई करना चाहते हैं?
-item.view.open मॉडल व्यूअर खोलें
-item.view.more अधिक जानकारी
-item.sell.cant आइटम बेचा नहीं जा सकता
-item.sell.quick क्या आप वास्तव में {0} की शीघ्र बिक्री करना चाहते है?
-social.profile.stats संपूर्ण आँकड़े
-social.profile.class हथियार के आँकड़े
-social.profile.sales बिक्री
-social.profile.sales.market मार्केट में बिक्री
-social.profile.skins स्किन्स [{0}]
-social.profile.skins.none उपयोगकर्ता द्वारा कोइ भी स्किन नही
-social.profile.games खेल [{0}]
-social.profile.games.none उपयोगकर्ता द्वारा कोइ भी खेल नही
-social.profile.assets एसेट्स [{0}]
-social.profile.assets.none उपयोगकर्ता द्वारा कोइ भी एसेट्स नही
-social.profile.mods मॉड्स [{0}]
-social.profile.mods.none उपयोगकर्ता द्वारा कोइ भी मॉड्स नही
-social.search खोज के परिणाम
-social.market.stats.rare सबसे दुर्लभ वस्तु
-social.market.stats.circulation सर्कुलेशन मे
-social.market.stats.units दुनिया में कुल युनिट्स
-social.trades.in प्राप्त
-social.trades.in.head प्राप्त ट्रेड्स ({0})
-social.trades.out भेजे गए
-social.trades.out.head भेजी गयी ट्रेड्स ({0})
-social.trades.history इतिहास
-social.trades.history.head व्यापार का इतिहास - ({0})
-social.trades.decline सभी ट्रेड्स को खारिज करे
-social.trades.message कृपया सेटिंग से व्यापार संदेश सक्षम करें
-social.trades.warning आपको इस व्यापार से कोइ वस्तु नही मिलेगा!
-social.trades.accept क्या आप वास्तव में इस ट्रेड को स्वीकार करना चाहते हैं?
-stats.airdrops KR पैकेज
-settings.quality.flapAnim स्नाइपर फ्लैप एनीमेशन
+server.config.scoreLimit स्कोर सीमा
+server.config.keepTScore दल का स्कोर रखे
+server.config.forceC Force Mode Classes
+server.config.logTim समय का पता (दौड़,पारकोर)
+server.config.hitBoxPad हिटबॉक्स पैडिंग
+server.config.hpRegen स्वास्थ्य पुनर्जन्म
+server.config.disableB सीमाओं को निष्क्रिय करें
+server.config.throwMel फेंकने योग्य चाकु
+server.config.noCosm काज़्मेटिक
+server.config.tstCmp जाँच मोड
+server.config.limitClasses क्लास लिमिट करें
+server.config.tmSize टीम साइज़
+server.config.fallDmg फॉल डैमेज 
+server.config.fallDmgThr फॉल डैमेज thresh 

```

Comment:
Replace "thresh" with "की सीमा" as to make it "फॉल डैमेज की सीमा".
"thresh" seems abrupt and meaningless.